client/login/wp-login/style.scss

```diff
@@ -386,10 +386,15 @@ $image-height: 47px;
 				background-color: #fdfdfd;
 				border: 0;
 				height: unset;
-				width: 194px;
 				text-align: start;
 			}
 		}
+
+		.login__form-userdata {
+			button {
+				width: 194px;
```

Comment:
What if we give the buttons min-width instead to allow it to grow?

Reply:
I can do that, it shouldn't cause any problem. This 194px for `login__form-userdata` was already there before I started working on this 👍

Reply:
Fixed!